scripts/update_avalanchego_version.sh

```diff
@@ -17,7 +22,7 @@ if [[ -n "${VERSION}" ]]; then
 fi
 
 # Discover AVALANCHE_VERSION
-. scripts/versions.sh
+. "$SUBNET_EVM_PATH"/scripts/constants.sh
```

Comment:
(Apologies for my lack of shell knowledge!)

Couldn't this work as before with `. scripts/constants.sh`?

Reply:
I think that would work, but better to use the full path.